Display/features/gets_sensor_data.feature

```diff
@@ -0,0 +1,15 @@
+Feature: gets data from sensors
+
+    Scenario: connects to the sensors that are connected
+        Given some sensors are physically connected to the Pressure Sensor and Airflow Sensor PI Hat
+        Then the software will determine which tubes have both pressure and airflow sensing
+        Then the software will initialize and calibrate the sensors
+        Then the software will return the state of the sensor to the user.
+
+    Scenario: there are not enough sensors
+        Given there are no tubes with both pressure and airflow sensing connected
```

Comment:
- Given a set of functioning sensors
- and a set of malfunctioning sensors or parts
- when the software diagnostic is run
- then this error will be raised

With the above approach, you can actually have a pretty complete set of tests to say things like "Flow sensor 2 appears to be malfunctioning", nomsain?